WebRoot/css/zm.css

```diff
@@ -3906,15 +3906,43 @@ HTML>BODY .appt-selected .appt_allday_body
 	height:auto;
 }
 
-.ZmPrefZimletListView .DwtListView-Rows {
-	overflow:visible;
+.ZmPrefZimletDetail {
+	@PrefZimletDetail@
 }
 
-.ZmPrefZimletListView .Row {
-	overflow:hidden;
+.ZOptionsItemsListView {
+	@PanelBorder@
+}
+
+.ZOptionsItemsListView .DwtListView-ColHeader {
+	@OptionsItemsListColumnHeaderWrapper@
 }
 
-.ZmPrefZimletListView .Row-selected {
+.ZOptionsItemsListView .DwtListView-Column {
+	@OptionsItemsListColumnHeader@
```

Comment:
property name is ColumnHeader but class is Column, is this for header or all cells in column?